src/sass/extends/_card-image-and-content.scss

```diff
@@ -27,11 +27,13 @@
     flex-direction: row;
 
     .content {
+      width: 55%;
       padding: $uds-size-spacing-4;
     }
 
     .card {
-      margin: -201px 96px 0 96px;
+      width: 35%;
+      margin: -201px 5% 0 10%;
```

Comment:
This seems like a really bad approach for building this element. LOL. 

We do not have this element implemented in the theme, and if we DO decide to do it, I'll rewrite it so that it's more solid. I DO like the design aesthetic here. 

Link: https://unity.web.asu.edu/@asu-design-system/bootstrap4-theme/index.html?path=/story/components-content-sections-card-image-and-content--default